.azure-devops/create-release.yml

```diff
@@ -97,24 +97,33 @@ stages:
       displayName: 'Test DPS'
       condition: eq('${{ parameters.testDPS }}', true)
       steps:
-      - template: templates/run-tests.yml
+      - template: templates/run-tests-parallel.yml
         parameters:
           path: 'azext_iot/tests/dps'
           name: 'device-provisioning-service'
     
-    - job: 'testHub'
-      displayName: 'Test IoT Hub'
+    - job: 'testHub_job_1'
+      displayName: 'Test IoT Hub - config, core and jobs'
       condition: eq('${{ parameters.testHub }}', true)
       steps:
-      - template: templates/run-tests.yml
+      - template: templates/run-tests-parallel.yml
         parameters:
-          path: 'azext_iot/tests/iothub'
+          path: 'azext_iot/tests/iothub/configurations azext_iot/tests/iothub/core azext_iot/tests/iothub/jobs'
+          name: 'iot-hub'
```

Comment:
This needs a different name to not conflict with distributed code coverage combining